tools/cli/go-whisk/whisk/client.go

```diff
@@ -293,7 +297,12 @@ func (c *Client) Do(req *http.Request, v interface{}) (*http.Response, error) {
     // Handle 1. HTTP Success + Valid body matching request expectations
     // Handle 3. HTTP Success + Body does NOT match request expectations
     if IsHttpRespSuccess(resp) && v != nil {
-        return parseSuccessResponse(resp, data, v), nil
+        if processTimeout && resp.StatusCode == EXITCODE_TIMED_OUT {
+            err = MakeWskError(errors.New("Time limit exceeded."), EXITCODE_TIMED_OUT, NO_DISPLAY_MSG, NO_DISPLAY_USAGE,
```

Comment:
- Globalize message.
- I think this message will need to be a bit more descriptive.. That is, the corresponding request is still being processed and that the activation record must be queried to see the final results of the request... but in a more pithy message :-)
- Oh.. I see the `NO_DISPLAY_MSG` flag here.. Per above bullet, I think we need to tell the user something more than just dumping the activation json object..